### ISSUE TITLE
The 'title' prop of the DateRangePicker is not readable by the screen readers

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-vue#248]
**Describe the bug**
When the 'title' property of the DateRangePicker is defined for the component, its value is not readable by the screen readers. 

**To Reproduce**
1. Open [this StackBlitz example](https://stackblitz.com/edit/wg46ys-do3vae?file=src%2Fmain.vue)
2. Start a screen reader of your choice
3. Using the 'Tab' key navigate through the DateRangePicker 

**Expected behavior**
At some point, the value of the 'title' prop should be announced by the screen readers

**Comment by zdravkov** [comment]
Since we have two inputs in this component, when title is needed we rather recommend using the startDateInputSettings and endDateInputSetttings props to set the desired titles as for example it is done here - https://stackblitz.com/edit/wg46ys-uav1zz?file=src%2Fmain.vue
